Yosemite/Yosemite/PointOfSale/POSProductProvider.swift

```diff
@@ -66,20 +67,31 @@ final class POSProductProvider {
 // MARK: - PreviewProvider helpers
 //
 extension POSProductProvider {
-    static func provideProductForPreview(currencySettings: CurrencySettings = ServiceLocator.currencySettings) -> POSProduct {
+    public static func provideProductForPreview() -> POSProduct {
         POSProduct(itemID: UUID(),
                    productID: 1,
                    name: "Product 1",
                    price: "1.00",
-                   currencySettings: currencySettings)
+                   formattedPrice: "$1.00")
     }
 
-    static func provideProductsForPreview(currencySettings: CurrencySettings = ServiceLocator.currencySettings) -> [POSProduct] {
+    public static func provideProductsForPreview() -> [POSProduct] {
```

Comment:
❓ Now that we have a protocol `POSItemProvider` and`POSItem` and this is just for SwiftUI previews, how about creating another implementation of `POSItemProvider` & `POSItem` within DEBUG in the app layer to provide mock items for previews?